src/PeriodDuration.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spatie\Period;
+
+use DateTimeImmutable;
+
+final class PeriodDuration
```

Comment:
```suggestion
class PeriodDuration
```

You said in https://github.com/spatie/period/pull/17#issuecomment-457471043 that you'd like the classes in this library to be extensible 🙈